pkg/arvo/lib/hood/drum.hoon

```diff
@@ -329,23 +330,66 @@
   [%give %fact ~[/drum] %dill-blit !>(dill-blit)]
 ::
 ++  se-adit                                           ::  update servers
-  ^+  .
-  ::  ensure dojo connects after talk
-  =*  dojo-on-top  |=([a=* b=*] |(=(%dojo a) &(!=(%dojo b) (aor a b))))
-  %+  roll  (sort ~(tap in ray) dojo-on-top)
-  =<  .(con +>)
-  |:  $:{wel/well:gall con/_..se-adit}  ^+  con
-  =.  +>.$  con
-  =+  hig=(~(get by fur) q.wel)
-  ?:  &(?=(^ hig) |(?=(~ u.hig) =(p.wel syd.u.u.hig)))  +>.$
-  =.  +>.$  (se-text "activated app {(trip p.wel)}/{(trip q.wel)}")
-  %-  se-emit(fur (~(put by fur) q.wel ~))
+  ^+  this
+  |^
+  =/  servers=(list well:gall)
+    (sort ~(tap in ray) sort-by-priorities)
+  |-
+  ?~  servers
+    this
+  =/  wel=well:gall
+    i.servers
   =/  =wire  [%drum p.wel q.wel ~]
-  [%pass wire %arvo %g %conf [our.hid q.wel] our.hid p.wel]
+  =/  hig=(unit (unit server))
+    (~(get by fur) q.wel)
+  ?:  &(?=(^ hig) |(?=(~ u.hig) =(p.wel syd.u.u.hig)))
+    this
+  =.  fur
+    (~(put by fur) q.wel ~)
+  =.  this
+    (se-text "activated app {(trip p.wel)}/{(trip q.wel)}")
+  =.  this
+    %-  se-emit
+    [%pass wire %arvo %g %conf [our.hid q.wel] our.hid p.wel]
+  $(servers t.servers)
+  ::
+  ++  priorities
+    ^-  (list (set @))
+    :~
+      (sy ~[%dojo])  ::  ensure dojo connects first
+      %-  sy
+      :~  %permission-store
```

Comment:
Maybe:

```suggestion
      ::  ensure dojo connects first
      ::
      (sy ~[%dojo])
    ::
      ::  set up stores with priority: depended on, but never depending
      ::
      %-  sy
      :~  %permission-store
```

We also sort for dojo below though, I think this is just boot order rather than connect order, which for dojo doesn't really matter?